docs/index.rst

```diff
@@ -3,7 +3,7 @@
 Contents:
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
```

Comment:
A work around for lists not rendering properly with the theme (all in the same level)